src/library/scala/collection/mutable/ArrayBuffer.scala

```diff
@@ -57,7 +57,12 @@ class ArrayBuffer[A] private (initialElements: Array[AnyRef], initialSize: Int)
 
   /** Reduce length to `n`, nulling out all dropped elements */
```

Comment:
The comment should reflect under which conditions the array is reallocated instead of nulling it.